wordpress.org/public_html/wp-content/themes/pub/wporg-support/functions.php

```diff
@@ -228,23 +394,22 @@ function wporg_support_get_views() {
 		'all-topics',
 		'no-replies',
 		'support-forum-no',
-		'taggedmodlook',
```

Comment:
Why is this removed?

Reply:
It isn't in the design, but I guess that's a logged out view. Reverted in [2e5f3f8](https://github.com/WordPress/wordpress.org/pull/201/commits/2e5f3f8f3b14d4b90be8a32acd7d7645d1fb3efb)